pkg/cmd/agent-task/create/create.go

```diff
@@ -29,12 +30,14 @@ type CreateOptions struct {
 	ProblemStatement string
 	BackOff          backoff.BackOff
 	BaseBranch       string
+	Prompter         prompter.Prompter
 }
 
 func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Command {
 	opts := &CreateOptions{
 		IO:         f.IOStreams,
 		CapiClient: shared.CapiClientFunc(f),
+		Prompter:   f.Prompter,
```

Comment:
We can also add assign `Config` here:


```suggestion
		Prompter:   f.Prompter,
        Config:     f.Config,
```

Reply:
Done.